Migration.Executor.WebJob/ChangeFeedProcessorHost.cs

```diff
@@ -203,28 +206,45 @@ private async Task ProcessChangesAsync(IReadOnlyCollection<DocumentMetadata> doc
         {
             try
             {
-                Boolean isSyntheticKey = this.SourcePartitionKeys.Contains(",");
-                Boolean isNestedAttribute = this.SourcePartitionKeys.Contains("/");
+                Boolean isSyntheticKey = this.SourcePartitionKey.Contains(",");
+                Boolean isNestedAttribute = this.SourcePartitionKey.Contains("/");
                 Container targetContainer = this.destinationCollectionClient.GetContainer(this.config.DestDbName, this.config.DestCollectionName);
                 this.containerToStoreDocuments = targetContainer;
-                DocumentMetadata document;
                 BulkOperations<DocumentMetadata> bulkOperations = new BulkOperations<DocumentMetadata>(docs.Count);
                 foreach (DocumentMetadata doc in docs)
                 {
-                    document = (this.SourcePartitionKeys != null & this.TargetPartitionKey != null) ?
-                        MapPartitionKey(doc, isSyntheticKey, this.TargetPartitionKey, isNestedAttribute, this.SourcePartitionKeys) :
-                        document = doc;
-                    if (this.config.OnlyInsertMissingItems)
+
+                    if (this.SourcePartitionKey != null)
                     {
-                        bulkOperations.Tasks.Add(this.containerToStoreDocuments.CreateItemAsync(
-                            item: document,
-                            cancellationToken: cancellationToken).CaptureOperationResponse(document, ignoreConflicts: true));
+                        doc.PK = this.TargetPartitionKey != null && this.SourcePartitionKey != this.TargetPartitionKey
```

Comment:
Is this PK property serialized? How does it map to the different property name if the destination has a different path / synthetic key?